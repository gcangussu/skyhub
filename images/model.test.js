const Image = require('./model');

test('Image model', () => {
  const image = new Image({
    name: 'test',
    small: 'test_samall.png',
    medium: 'medium_test.jpg',
    large: 'large.gif',
  });
  const imageObj = image.toObject();
  /* eslint no-underscore-dangle: "off" */
  delete imageObj._id;
  expect(imageObj).toMatchSnapshot();
});
