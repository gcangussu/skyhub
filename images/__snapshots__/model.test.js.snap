// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Image model 1`] = `
Object {
  "large": "large.gif",
  "medium": "medium_test.jpg",
  "name": "test",
  "small": "test_samall.png",
}
`;
